app/controllers/posts_controller.rb

```diff
@@ -6,5 +6,26 @@ def index
 
   def show
     @post = Post.find(params[:id])
+    @current_user = User.first
+    @like = Like.new
+  end
+
+  def new
+    @post = Post.new
+  end
+
+  def create
+    @post = current_user.posts.build(post_params)
+    if @post.save
+      redirect_to user_posts_path(current_user), notice: 'Post created successfully!'
+    else
+      render :new
+    end
+  end
```

Comment:
- [ ] When I try to create a `Post`, I'm getting the following errors:

![image](https://github.com/ankitt26/Ankye-blogs/assets/77069011/8b7521ec-9816-4be7-8dde-0136a1c101ec)

Please consider fixing them! 🤓 🔧 

💡 _Hint: You could run a migration to set the default values of these counters to 0, so they become optional parameters._

Reply:
Thanks, I will fix that.

Reply:
Well resolved ✔️ 

Posts, Comments, and Likes can be added succesfully. 🚀 

![image](https://github.com/ankitt26/Ankye-blogs/assets/38088818/c43fa374-24a4-4928-af91-194fed3d366c)